Vostok.Hosting.Abstractions/IVostokHostExtensions.cs

```diff
@@ -36,5 +36,10 @@ public interface IVostokHostExtensions
         /// Returns all types registered without key.
         /// </summary>
         IEnumerable<(Type, object)> GetAll();
+        
+        /// <summary>
+        /// Returns all types registered.
+        /// </summary>
+        IEnumerable<(Type Type, object Extension, string Key)> GetAll(bool withoutKeys, bool withKeys);
```

Comment:
You can't extend interfaces this way, it breaks backward compatibility

Reply:
Still actual

Reply:
Where you found it?